apps/mobile/src/features/receive/receive-sheets/select-asset.tsx

```diff
@@ -93,7 +93,7 @@ export function SelectAsset() {
           />
         }
       >
-        <ScrollView>
+        <ScrollView scrollEnabled={assets.length > 9}>
```

Comment:
I added this check so it only scrolls if the screen is full. It's easy to check on this screen but on `SelectAssetSheet` it's more difficult as the `ScrollView` wraps loaders that load account assets. 

```
  <ScrollView>
        <BitcoinPayerLoader fingerprint={account.fingerprint} accountIndex={account.accountIndex}>
          {({ nativeSegwitPayer }) => (
            <BitcoinBalanceByAccount
    ...
```

Maybe there's a better way of using `ScrollView` to only enable scroll if the screen is full? I was looking at [alwaysBounceVertical](https://reactnative.dev/docs/scrollview#alwaysbouncevertical-ios) but it seems to be iOS only

Reply:
@pete-watters is this helpful? I set this up to detect the height and enable/disable based on that...
https://github.com/leather-io/mono/blob/3068cea3905eae77aafbe890d802aee60d3ef919/apps/mobile/src/components/headers/animated-header/animated-header-screen.layout.tsx#L43